### ISSUE TITLE
[Question] About LoRA

### ISSUE THREAD

**Issue by Huli-fox** [ace-step/ACE-Step-1.5#737]
1.According to [this readme](https://github.com/koda-dernet/Side-Step?tab=readme-ov-file#technical-notes-timestep-sampling) , I would like to know if you consider it a bug and if it has been fixed in the latest version
> Technical Notes: Timestep Sampling
> Side-Step ensures your fine-tuning matches the base model's original training distribution:
> 
> 1. Turbo Models: Uses discrete 8-step sampling (matching inference).
> 2. Base/SFT Models: Uses continuous logit-normal sampling + CFG Dropout (matching training).
> 
> The upstream trainer often forces the Turbo schedule on all models, which is incorrect for Base/SFT. Side-Step fixes this automatically.

2.About [this](https://github.com/ace-step/ACE-Step-1.5/blob/main/docs/en/LoRA_Training_Tutorial.md#using-lora), 
I tried loading Lora directly without restarting, the generated audio was somewhat abnormal. I think this is a bug and I would like to know if it can be fixed to improve the user experience
Does this mean that if multiple datasets need to be trained, a restart is required after each training?
> After training completes, restart Gradio...

**Comment by 1larity** [comment]
Side-step is an independent project, please raise site-step issues on the appropriate [repo](https://github.com/koda-dernet/Side-Step)

**Comment by Huli-fox** [comment]
For those who come later:
Q1.The developers wont support train on base/sft with Gradio, you need to use sidestep.
Q2.The developers didnt respond.
(According to the developer's response on Discord)

**Comment by ChuxiJ** [comment]
@Huli-fox 

<img width="1708" height="1588" alt="Image" src="https://github.com/user-attachments/assets/7cad3b02-5c92-49f6-a35e-2a9136afd3b2" />

**Comment by ChuxiJ** [comment]
The key point here is that you can submit a PR to support **SFT training with more timestep sampling** in Gradio.
But I think Gradio only provides a simple training method.
If you want more advanced training, you can use the **side-step** project instead.

**Comment by Huli-fox** [comment]
> The key point here is that you can submit a PR to support **SFT training with more timestep sampling** in Gradio. But I think Gradio only provides a simple training method. If you want more advanced training, you can use the **side-step** project instead.

@ChuxiJ Sorry, I didn't convey that quite accurately.

**Comment by Huli-fox** [comment]
So Q2 still hasn't been answered. This question came up when I was using Gradio before I learned about sidestep, and it has nothing to do with sidestep. I'm not sure if this is due to limitation of Gradio itself or something.

**Comment by ChuxiJ** [comment]
@Huli-fox Just restart and load again.
The environment and model have changed.
If you find restarting inconvenient, you can submit a PR to load the LoRA directly after training without restarting the demo.
